src/data.js

```diff
@@ -10596,6 +10596,35 @@ module.exports = [
       'SQL',
     ],
   },
+  {
+    name: 'Sahilpreet Singh',
+    description: 'MERN stack developer, Leaning Devops, Web Enthusiast.',
+    url: 'https://github.com/preetsahil/uses',
+    twitter: '@sahil21_7',
+    emoji: '😏',
+    country: '🇮🇳',
+    computer: 'windows',
+    phone: 'OPPO RENO 3 PRO',
```

Comment:
phone options should be one of the following: 
```
[iphone, android, windowsphone, flipphone]
```

Reply:
making chnages.